packages/elements/src/components/ino-carousel/ino-carousel.e2e.ts

```diff
@@ -53,7 +53,7 @@ describe('InoCarousel', () => {
     </ino-carousel>
     `);
     inoCarousel = await page.find('ino-carousel');
-    inoCarouselWrapper = await page.find('ino-carousel > div');
+    //inoCarouselWrapper = await page.find('ino-carousel > div');
```

Comment:
Why you comment this out?

Reply:
I got some error from this line that wouldn't let me run the code, so i commented them out. I just put it back in, seems to run without any problems... so idk what happened there 🤔🤷‍♀️